config/locales/en.yml

```diff
@@ -1181,6 +1181,8 @@ en:
     edit:
       add_keyword: Add keyword
       keywords: Keywords
+      statuses: Individual posts
+      statuses_hint_html: This filter applies to select individual posts regardless of whether they match the keywords below. You can review these posts and remove them from the filter by <a href="%{path}">clicking here</a>.
```

Comment:
I think "click here" links are considered an anti-pattern. Could be reworded to just be "Review and remove posts from the filter"